rlbot.cfg

```diff
@@ -7,7 +7,7 @@
 [Match Configuration]
 # Visit https://github.com/RLBot/RLBot/wiki/Config-File-Documentation to see what you can put here.
 # Number of bots/players which will be spawned.  We support up to max 64.
-num_participants = 2
+num_participants = 1
 game_mode = Soccer
 game_map = Mannfield
 enable_rendering = True

```

Comment:
networking crashes for now if we add more than 1 bot because they all try to start a server and a client, and on the same host and port. To add more bots, we need to make only 1 bot handle everything, and make the rest do nothing, and return immediately in the main